astropy/modeling/blackbody.py

```diff
@@ -67,6 +67,10 @@ class BlackBody1D(Fittable1DModel):
     .. plot::
         :include-source:
 
+        # Ignore warnings about deprecation of BlackBody1D
+        import warnings
+        warnings.simplefilter('ignore')
```

Comment:
Can we not fix the underlying cause? (The imporf of `FLAM`.)

Reply:
I wouldn't spend too much time on this module, as it is deprecated and will be removed in the future.

Reply:
Ah, sorry, I didn't see this was also in `blackbody.py` - yes, definitely no problem!